### ISSUE TITLE
Notes on S-corp, C-corp, LLC

### ISSUE THREAD

**Issue by steakknife** [leonar15/startup-checklist#4]
Nice work!

S-corp, C-corp and LLC/LLP are merely tax classifications ([IRS form 8832](https://www.irs.gov/uac/form-8832-entity-classification-election), [pdf](https://www.irs.gov/pub/irs-pdf/f8832.pdf)).

S-corps can always be converted into C-corps.

S-corps are usually less overhead, but VC's typically only invest in C-corps.

LLC/LLP are preferred for pure service / consulting companies that don't need investment.

And, here's some other resources:
- [clerky](https://www.clerky.com/)
- [Orrick, Herrington & Sutcliffe](https://www.orrick.com/) (a-k-a Orrick)
- [Wilson Sonsini Goodrich & Rosati](https://www.wsgr.com/) (a-k-a Wilson Sonsini, WSGR)

**Comment by jamiesonbecker** [comment]
@steakknife is correct. LLC/LLP's are basically impossible to convert as-is (ie requires a new company to be incorporated.) S-corp's are simple to convert to C-corp's, but the reverse is not true.

The reason VC's will not invest in subchapter S Corporations is chiefly because they can't: a venture capital firm is not a natural person, and S Corps can only have natural persons as shareholders (not other corporations). There are tax benefits to C corps as well in the early years for VCs. However, S-corps in your state are much more favorable for bootstrapping.

It's not possible to convert an S-corp in your state to a Delaware C-corp directly, so the choices that are made early on can have a significant, expensive impact if you change direction later.

IANAL

**Comment by steakknife** [comment]
Here's some add'l docs on DE incorporation conversions and S-corps:

http://www.nolo.com/legal-encyclopedia/converting-llc-corporation-s-corporation-delaware.html

http://www.nolo.com/legal-encyclopedia/converting-corporation-s-corporation-llc-delaware.html

http://www.nolo.com/legal-encyclopedia/corporations-vs-llcs-29025.html

**TL;DR - Startup? Stick with a DE C-corp.**

Don't incorporate until absolutely needed or are about to take on risks which would require limited liability / insurance.  It's usually unnecessary until significant money or investment is coming in.  Screwing up incorporation can be very expensive (legal bills) and potentially turn off investment.

But hey, if you just need an DE LLC quick and cheap, I've used [incnow](https://www.incnow.com/) for enterprise startup consulting.

**Comment by jamiesonbecker** [comment]
"TL;DR: Stick with a DE C-corp." great advice unless you specifically know that you need something else, just do that.

**Comment by jamiesonbecker** [comment]
Also, https://stripe.com/atlas

**Comment by steakknife** [comment]
Nice. Hopefully Stripe makes incorporation in the UK easier, because it's [laborious](https://www.gov.uk/topic/company-registration-filing/starting-company) (tracking down some lady in London whom literally sells corporate registers out of the back of her panel van because reputable legal bookstores no longer stock them for no given reason).
